novawallet/Modules/Transfer/BaseTransfer/OnChain/OnChainTransferPresenter.swift

```diff
@@ -113,18 +123,18 @@ class OnChainTransferPresenter {
             ),
 
             dataValidatingFactory.canPayFeeSpendingAmountInPlank(
-                balance: senderUtilityAssetTransferable,
+                balance: senderAssetTransferable,
                 fee: fee?.value,
-                spendingAmount: isUtilityTransfer ? sendingAmount : nil,
-                asset: utilityAssetInfo,
+                spendingAmount: feeAsset == chainAsset ? sendingAmount : nil,
```

Comment:
it is safer to compare chainAssetId as it never changes

Reply:
Yep, updated